src/components/Store/StoreContainer.tsx

```diff
@@ -33,9 +33,17 @@ export const StoreContainer = () => {
     category: searchParams.get('category') || '',
     descr: searchParams.get('descr') || '',
     rating: searchParams.get('rating') || '',
-    stock: searchParams.get('rating') || '',
+    stock: searchParams.get('stock') || '',
+    price: searchParams.get('price') || '',
   };
 
+  /*   const searchNameParam = searchParams.get('name');
+  const searchBrandParam = searchParams.get('brand');
+  const searchCategoryParam = searchParams.get('category');
+  const searchDescrParam = searchParams.get('descr');
+  const searchRatingParam = searchParams.get('rating');
+  const searchStockParam = searchParams.get('stock'); */
+
```

Comment:
Is it necessary or you can remove it?

Reply:
> Is it necessary or you can remove it?

I don't think we'll need it in the future. Our data is localized in the application itself, that is, after changing these parameters, we do not need to make a request to the server again with the parameters already changed in the Url request.